paths/private_groups.yaml

```diff
@@ -24,21 +27,16 @@ get:
         enum:
           - created_at
           - id
-    - name: active
-      in: query
-      description: return either active of inactive private groups
-      required: false
-      schema:
-        type: boolean
+    - $ref: '../parameters/OrderDir.yaml'
     - $ref: '../parameters/Schema.yaml'
-    - $ref: '../parameters/X-User-Token-Optional.yaml'
     - $ref: '../parameters/Bearer.yaml'
```

Comment:
_technically_ all requests would need this parameter, except a handful of requests to generate that token

Reply:
I was thinking whether I should add this for all endpoints too, what do you think?

Reply:
it's a bit of annoying work, but it would technically be correct, because otherwise none of the endpoints would work